Controller/Process/Result.php

```diff
@@ -69,9 +68,9 @@ class Result extends \Magento\Framework\App\Action\Action
     protected $quoteFactory;
```

Comment:
I think you can remove `$quoteFactory` as well

Reply:
You're right!